plugins/server/io.ktor/ktor-di/versions.ktor.yaml

```
@@ -0,0 +1,3 @@
+# This file maps ktor versions to the required artifacts
+# See templates/versions.ktor.yaml for more details
+"[3.2,)": ktor-server-di:==
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)